boms/cloud-oss-bom/pom.xml

```diff
@@ -50,11 +50,6 @@
         <artifactId>guava</artifactId>
         <version>${guava.version}</version>
       </dependency>
-      <dependency>
```

Comment:
Maybe comment this out instead of deleting it because it's a short-term hack to remove it?

Reply:
done